hypervisor/include/hw/pci.h

```diff
@@ -43,6 +43,8 @@
  * PCIZ_xxx: extended capability identification number
  */
 
+#define PCI_STD_HEADER_LENGTH 0x40U
```

Comment:
PCI_HEADER_LENGTH ?  
I didn't see STD in spec.

Reply:
Will rename it